frame/curve-amm/src/lib.rs

```diff
@@ -133,30 +144,31 @@ pub mod pallet {
 	/// Current number of pools (also ID for the next created pool)
 	#[pallet::storage]
 	#[pallet::getter(fn pool_count)]
-	pub type PoolCount<T: Config> = StorageValue<_, PoolId, ValueQuery>;
+	pub type PoolCount<T: Config> = StorageValue<_, T::PoolId, ValueQuery>;
 
 	/// Existing pools
 	#[pallet::storage]
 	#[pallet::getter(fn pools)]
-	pub type Pools<T: Config> = StorageMap<_, Blake2_128Concat, PoolId, PoolInfo<T::AccountId>>;
+	pub type Pools<T: Config> =
+		StorageMap<_, Blake2_128Concat, T::PoolId, StableSwapPoolInfo<T::AccountId>>;
 
 	/// Pool's LP asset
 	#[pallet::storage]
 	#[pallet::getter(fn pool_lp_asset)]
-	pub type PoolLPAsset<T: Config> = StorageMap<_, Blake2_128Concat, PoolId, T::AssetId>;
+	pub type PoolLPAsset<T: Config> = StorageMap<_, Blake2_128Concat, T::PoolId, T::AssetId>;
 
 	/// List of assets supported by the pool
 	#[pallet::storage]
 	#[pallet::getter(fn pool_assets)]
-	pub type PoolAssets<T: Config> = StorageMap<_, Blake2_128Concat, PoolId, Vec<T::AssetId>>;
+	pub type PoolAssets<T: Config> = StorageMap<_, Blake2_128Concat, T::PoolId, Vec<T::AssetId>>;
```

Comment:
could you leave comment if other pools on substratre also have Vec, not BoundedVec in list?

Reply:
I think https://github.com/equilibrium-eosdt/equilibrium-curve-amm/tree/master/pallets/equilibrium-curve-amm/src